src/shared_modules/content_manager/tests/unit/offlineDownloader_test.cpp

```diff
@@ -156,3 +156,22 @@ TEST_F(OfflineDownloaderTest, TwoFileDownloadsOverrideOutput)
     EXPECT_EQ(m_spUpdaterContext->data, expectedData);
     EXPECT_TRUE(std::filesystem::exists(m_spUpdaterBaseContext->contentsFolder / m_inputFilePathRaw.filename()));
 }
+
+/**
+ * @brief Tests the download of a raw file with an inexistant content folder. Exception is expected.
+ *
+ */
+TEST_F(OfflineDownloaderTest, InexistantContentFolder)
```

Comment:
Note: This test is added to cover the exception test case that, given the new changes, was no longer covered.